### PR TITLE
AF-354 update readme, prepare for public app

### PR DESCRIPTION
**Pull request by sandlerr**
@zendesk/vegemite 

### Description

- updated node version as I refuse to work on any project on node <=6.0 except for one special project
- added a Procfile with _optional_ foreman instructions
- `private: false`
- actually write a readme
- https://zendesk.atlassian.net/browse/AF-354

### CODE REVIEW

**Comment by ocke** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -1,2 +1,50 @@
...
+If you need to test a change to this app, you'll need to setup the dependencies by running:
+
+```
+gem install zendesk_apps_tools
+npm install
+npm install -g webpack # optionally install webpack so it can be run from the command-line
+npm install -g foreman # optionally install foreman to run both servers at once
+```
+
+To run the app, if you installed the optional dependencies you can run `nf start`.
```
Comment (svizzari) [inline README.md]: Should mention that foreman needs to be run from the root of the app folder structure, not the `dist/` folder?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,50 @@
...
+# iFrame REPL App
+
+## Description
+An interactive programming environment for ZAF SDK v2 developers.
+
+## Owners
+This repo is maintained by @zendesk/vegemite. You can contact us using that Github handle on issues or pull requests in this repo.
+We are often active during daytime hours on the east coast of Australia (UTC+10).
+
+We have a Slack channel called `vegemite` accessible to all Zendesk employees. The broader developer community is encourage to get in
```
Comment (svizzari) [inline README.md]: @pmgrove and I discussed putting a link to the beta signup form in here too

File: README.md (hunk 1 above)
Comment (sandlerr) [inline README.md]: all commands that I documented are run from the root. Wouldn't that be a sensible implicit assumption?

File: README.md (hunk 2 above)
Comment (sandlerr) [inline README.md]: what conclusion did you arrive at?

Comment (danielbreves) [inline README.md]: > encourage**d**

**Comment by danielbreves** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (svizzari) [inline README.md]: you know what they say about assumptions

File: README.md (hunk 2 above)
Comment (svizzari) [inline README.md]: > what conclusion did you arrive at?

we should do it

File: README.md (hunk 1 above)
Comment (sandlerr) [inline README.md]: assuming you're referring to the popular saying, yes (will add a note)

**Comment by sandlerr** [comment]
@svizzari 💇

**Comment by svizzari** [comment]
👍

**Pull request merged**